specification/src/tutorial/explain.md

```diff
@@ -1,9 +1,11 @@
 # Explain
 
-The `/explain` endpoint is not implemented in the reference implementation, simply because tje `QueryResponse` is interpreted directly in the `/query` endpoint. There is no intermediate representation (such as SQL) which could be described as a "query plan".
+The `/query/explain` and `/mutation/explain` endpoints are not implemented in the reference implementation, simply because the `QueryResponse` is interpreted directly in the `/query` endpoint.
+There is no intermediate representation (such as SQL) which could be described as a "query plan".
```

Comment:
```suggestion
```suggestion
The `/query/explain` and `/mutation/explain` endpoints are not implemented in the reference implementation, because their respective request objects are interpreted directly. There is no intermediate representation (such as SQL) which could be described as an "execution plan".
```